runtime/lua/vim/filetype/detect.lua

```diff
@@ -619,6 +619,31 @@ function M.hw(_, bufnr)
   return 'virata'
 end
 
+-- This function checks for an assembly comment or a SWIG keyword or verbatim
+-- block in the first 50 lines.
+-- If not found, assume Progress.
+--- @type vim.filetype.mapfn
+function M.i(path, bufnr)
+  if vim.g.filetype_i then
+    return vim.g.filetype_i
+  end
+
+  -- These include the leading '%' sign
+  local ft_swig_keywords =
+    [[^\s*%\%(addmethods\|apply\|beginfile\|clear\|constant\|define\|echo\|enddef\|endoffile\|extend\|feature\|fragment\|ignore\|import\|importfile\|include\|includefile\|inline\|insert\|keyword\|module\|name\|namewarn\|native\|newobject\|parms\|pragma\|rename\|template\|typedef\|typemap\|types\|varargs\|warn\)]]
+  -- This is the start/end of a block that is copied literally to the processor file (C/C++)
+  local ft_swig_verbatim_block_start = '^%s*%%{'
+
+  for _, line in ipairs(getlines(bufnr, 1, 50)) do
+    if line:find('^%s*;') or line:find('^/%*') then
```

Comment:
```suggestion
    if line:find('^%s*;') or line:find('^%*') then
```

I missed this

Reply:
Yep, I just moved the function after the rename to keep things alphabetized.